### PR TITLE
Adding cloud native security whitepaper blog post with image

### PR DESCRIPTION
**Pull request by PushkarJ**
Hello!

This is a blog post that is a part of the CNCF SIG Security and Kubernetes SIG Security Docs Working Group collaboration. The post attempts to explain how the upcoming cloud native security whitepaper relates to securing a k8s cluster. It has been reviewed by @sftim. Tagging @onlydole as well, based on Tim's request.

We plan to publish this few hours after the whitepaper is made available to the public on November 18 2020. Once the permanent link to the whitepaper is available, will add new commit replacing the placeholder with the actual link to whitepaper as reference.

### CODE REVIEW

**Review by onlydole — APPROVED** [review]
Thank you for this excellent post, @PushkarJ!

**Review by PushkarJ — COMMENTED** [review]
/hold

File: content/en/blog/_posts/2020-11-18-cloud-native-security-for-your-cluster/index.md
Comment (sftim) [inline content/en/blog/_posts/2020-11-18-cloud-native-security-for-your-cluster/index.md:14]: _nit_: I'd use Markdown rather than HTML and a slightly different alt attribute
```suggestion
![Cloud native application lifecycle phases](cloud-native-app-lifecycle-phases.png)
```

**Comment by sftim** [comment]
If I get a chance to, I'll switch the image from PNG to SVG (I don't have the vector source yet though).

**Comment by PushkarJ** [comment]
> If I get a chance to, I'll switch the image from PNG to SVG (I don't have the vector source yet though).

@sftim: I have it, but I can not crop the huge white border for SVG, so cropped and used PNG. If you can, will DM you the svg on slack. Let me know, what you prefer.

**Comment by sftim** [comment]
> If you can, will DM you the svg on slack. Let me know, what you prefer.

I've done this before @PushkarJ  - might have to read up on https://css-tricks.com/scale-svg/ and edit some XML!

**Comment by sftim** [comment]
For an SVG image, check out commit [sftim@kubernetes-website:`ebd4091`](https://github.com/PushkarJ/website/compare/cncf-sig-security-whitepaper-k8s-blog-post...sftim:cncf-sig-security-whitepaper-k8s-blog-post-svg).
(feel free to cherry-pick that into this PR and then squash down).

I used an `` element to make the vector diagram render (a bit more) responsively between mobile and desktop style user agents.

**Comment by sftim** [comment]
Ready to LGTM when the whitepaper link is available.

[Preview](https://deploy-preview-25010--kubernetes-io-master-staging.netlify.app/blog/2020/11/18/cloud-native-security-for-your-clusters/)

**Pull request merged**
